.dotnet/api/api.md

```diff
@@ -291,7 +292,8 @@ namespace OpenAI.Assistants {
         public IList<string> FileIds { get; init; }
     }
     public class FileSearchToolDefinition : ToolDefinition {
-        public FileSearchToolDefinition();
+        public FileSearchToolDefinition(int? maxResults = null);
```

Comment:
we don't do property setters for ctor parameters

Reply:
I'll remove the constructor for the tactical update.

Other types we'll need to review for consistency:
- FunctionToolDefinition
- ToolOutput
- ChatFunction
- VectorStoreExpirationPolicy
-